pages.zh/common/cp.md

```diff
@@ -0,0 +1,28 @@
+# cp
+
+> 复制文件和文件夹。
+> 更多信息见：<https://www.gnu.org/software/coreutils/cp>.
```

Comment:
We use this translation for links:
```suggestion
> 更多信息：<https://www.gnu.org/software/coreutils/cp>.
```
I'm note sure about the `：` char, @bl-ue?

Reply:
Yep, the Chinese colon is the right one to use here.  And no space should come after it, I think.